test/integration/default/serverspec/elasticsearch.rb

```diff
@@ -38,6 +38,11 @@
     its(:stdout) { should match(/.*"status" : "green".*/) }
   end
 
+  # time to obey the previous command
+  describe command('sleep 10') do
```

Comment:
If really we need a sleep, I will put it in the describe that needs the sleep. This test on its is not really relevant :-(